dataclasses_json/mm.py

```diff
@@ -162,15 +163,16 @@ def __init__(self, *args, **kwargs):
             raise NotImplementedError()
 
         @typing.overload
-        def dump(self, obj: typing.List[A], many: typing.Optional[bool] = None) -> typing.List[TEncoded]:  # type: ignore
+        def dump(self, obj: typing.List[A],  # type: ignore
```

Comment:
Please revert code formatting changes, so the PR contains functional changes only :)